DIPs/1NNN-ADR.md

```diff
@@ -50,6 +50,9 @@ gets rewritten as:
 ```
 writefln(_d_interpolated_string!("I ate ", _d_interpolated_format_spec(null), " and ", _d_interpolated_format_spec("%d"), " totalling ", _d_interpolated_format_spec(null), " fruit.")(), apples, bananas, apples + bananas);
 ```
+
+where `_d_interpolated_string` and `_d_interpolated_format_spec` are defined exclusively inside druntime.
```

Comment:
What does this mean? That this won't work with better C at all?

Reply:
`string` is defined inside druntime and works inside betterC. This is no different.

This little addition just means that user code cannot redefine these templates to hook the language; all the rewrites in this document refer specifically to the one global definition, provided by druntime, rather than being looked up in local scope, potentially provided by a user.

Reply:
I just added more verbiage to try to clear that up a bit more, though I didn't mention betterC specifically, it'll work there. (my big concern here is to be very very clear to Walter that this is NOT "user-defined semantics")